scripts/release_sdk_status/main.py

```diff
@@ -209,9 +231,61 @@ def bot_analysis(self):
             self.bot_warning += 'Need to add track2 config.'
 
 
+def sdk_code_path(service_name, sdk_name) -> str:
+    return str(Path(os.getenv('SDK_REPO') + f'/sdk/{service_name}/{sdk_name}'))
+
+
+@return_origin_path
+def install_package_locally(service_name, sdk_name):
+    os.chdir(sdk_code_path(service_name, sdk_name))
+    print_check('pip install -e .')
+    print_exec('pip install -r dev_requirements.txt')
+
+
+@return_origin_path
+def run_test_proc(sdk_name, service_name, sdk_folder):
```

Comment:
next time please don't change code position for same function otherwise it is hard to review.